config/prow/cluster/boskos.yaml

```diff
@@ -0,0 +1,246 @@
+          jsonPath: .status.owner
+        - name: Last-Updated
+          type: date
+          jsonPath: .status.lastUpdate
+      schema:
+        openAPIV3Schema:
+          description: Abstracts any resource type that can be tracked by boskos
+          type: object
+          properties:
+            spec:
+              description: Holds information that are not likely to change
+              type: object
+              properties:
+                type:
+                  type: string
+            status:
+              description: Holds information that are likely to change
+              type: object
+              properties:
+                state:
+                  type: string
+                owner:
+                  type: string
+                lastUpdate:
+                  type: string
+                  format: date-time
+                userData:
+                  type: object
+                  x-kubernetes-preserve-unknown-fields: true
+                expirationDate:
+                  type: string
+                  format: date-time
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: boskos
+rules:
+- apiGroups: ["boskos.k8s.io"]
+  verbs: ["*"]
+  resources: ["*"]
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: boskos
+  namespace: test-pods
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: boskos
+subjects:
+- kind: ServiceAccount
+  name: boskos
+  namespace: test-pods
+roleRef:
+  kind: ClusterRole
+  name: boskos
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: boskos
+  namespace: test-pods
+spec:
+  replicas: 1  # one canonical source of resources
+  selector:
+    matchLabels:
+      app: boskos
+  template:
+    metadata:
+      labels:
+        app: boskos
+      namespace: test-pods
+    spec:
+      serviceAccountName: boskos
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: boskos
+        image: gcr.io/k8s-staging-boskos/boskos:v20211118-344faec
+        args:
+        - --config=/etc/config/boskos-resources.yaml
+        - --namespace=test-pods
+        ports:
+          - containerPort: 8080
+            protocol: TCP
+        volumeMounts:
+        - name: boskos-config
+          mountPath: /etc/config
+          readOnly: true
+      volumes:
+        - name: boskos-config
+          configMap:
+            name: resources
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: boskos
+  namespace: test-pods
+spec:
+  selector:
+    app: boskos
+  ports:
+  - name: default
+    protocol: TCP
+    port: 80
+    targetPort: 8080
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: boskos-metrics
+  namespace: test-pods
+spec:
+  selector:
+    app: boskos
+  ports:
+  - name: metrics
+    port: 9090
+    protocol: TCP
+    targetPort: 9090
+  loadBalancerIP: 104.197.27.114  # external ip boskos-metrics
+  type: LoadBalancer
```

Comment:
This is not required as of now, not sure if someone really need this metrics atm